optimade/filtertransformers/mongo.py

```diff
@@ -178,10 +178,6 @@ def constant_first_comparison(self, arg):
             for oper, prop in arg[1].items()
         }
 
-    def predicate_comparison(self, arg):
```

Comment:
So due to the discussion in [this PR](https://github.com/Materials-Consortia/OPTiMaDe/pull/191), which has still not been merged, and will maybe re-implement `predicate_comparison`, I am considering whether we should keep this in, calling it v0.10.0, or wait until the spec. PR has been merged, and rename this grammar to v0.10.1?

Or just accept the current complete discrepancy and "bug-fix" once the PR has been merged?

Reply:
I would say we could merge as it is because we need another PR anyway to implement all the functions of the transformer for all of the array related queries. I created a separate issue (#86 ) jut to dicuss the implementation of `LENGTH`.